packages/neuron-ui/package.json

```diff
@@ -7,7 +7,7 @@
     "email": "dev@nervos.org",
     "url": "https://github.com/nervosnetwork/neuron"
   },
-  "homepage": "./",
+  "homepage": "https://www.nervos.org",
```

Comment:
This field sets the root of the web app, not the homepage of the project.

Reply:
Sounds we can remove this line, no url for desktop app.

Reply:
True

Reply:
Not sure, the CRA fetch static files from `/` by default. And the Electron serves static files with the file system. Namely, the assets will be fetched from `file:///styles/xxx.css` which leads to errors. No verified. Please have a try.